example/resources/tags.js

```diff
@@ -4,7 +4,7 @@ jsonApi.define({
   namespace: "json:api",
   resource: "tags",
   description: "Used to group resources together, useful for finding related resources.",
-  handlers: jsonApi.mockHandlers,
+//  handlers: jsonApi.mockHandlers,
```

Comment:
Hmm you're right. How about we compromise, lets take the handler out completely for just one of the resources and fully replace the line with a comment pointing out that it's opting in to the default mockHandlers?